### PR TITLE
[ES] Add `osu! wiki`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_wiki/es.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Un artículo individual describe un solo término o concepto y ofrece una descripción general de temas adyacentes. Para ilustrar mejor el tema, se divide en subsecciones que cubren diferentes partes del término. También puede incluir capturas de pantalla relevantes u otras explicaciones visuales, o enlaces a otros artículos sobre el tema.
+
+Hay traducciones disponibles en diferentes idiomas para algunos artículos. Los crean los propios miembros de la comunidad y la elección del idioma solo está limitada por la [lista de idiomas](/wiki/Article_styling_criteria/Formatting#configuraciones-regionales) admitida por el propio sitio web.
+
+## Modelo
+
+La osu! wiki se basa en el concepto de accesibilidad: se recomiendan las ediciones y traducciones, que cualquier persona capaz y con conocimientos puede realizar. Los datos de los artículos se almacenan en un [repositorio de GitHub](https://github.com/ppy/osu-wiki) y los contribuidores los actualizan mediante el [uso de solicitudes de cambios](/wiki/osu!_wiki/Contribution_guide), que luego son revisados ​​y fusionados por los [mantenedores de la wiki](/wiki/People/osu!_wiki_maintainers).
```
Comment (agatemosu) [inline wiki/osu!_wiki/es.md:26]:
```suggestion
La osu! wiki se basa en el concepto de accesibilidad: se recomiendan las ediciones y traducciones, que cualquier persona capaz y con conocimientos puede realizar. Los datos de los artículos se almacenan en un [repositorio de GitHub](https://github.com/ppy/osu-wiki) y los contribuidores los actualizan mediante el [uso de solicitudes de cambios](/wiki/osu!_wiki/Contribution_guide), que luego son revisadas ​​y fusionadas por los [mantenedores de la wiki](/wiki/People/osu!_wiki_maintainers).
```

File: wiki/osu!_wiki/es.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Los artículos están formateados usando un [formato Markdown adaptado a GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/). Además, el motor del sitio web implementa características personalizadas cruciales para la wiki en su conjunto, como redireccionamientos, portada o mejoras de formato.
```
Comment (agatemosu) [inline wiki/osu!_wiki/es.md:28]: https://developer.mozilla.org/es/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN
```suggestion
Los artículos están formateados usando un [Markdown personalizado por GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/). Además, el motor del sitio web implementa características personalizadas cruciales para la wiki en su conjunto, como redireccionamientos, portada o mejoras de formato.
```

File: wiki/osu!_wiki/es.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- Se recomienda mejorar los artículos existentes y crear otros nuevos, pero es necesario un conocimiento básico de [GitHub](https://github.com) para poder contribuir, lo que eleva el umbral necesario para ingresar. Sin embargo, la wiki de osu! proporciona una [guía de contribución](/wiki/osu!_wiki/Contribution_guide) para ayudar a los nuevos contribuidores a empezar.
+- Todos los cambios en la wiki de osu! se moderan previamente y se someten a un proceso de revisión, en el que se espera que el contribuidor colabore con los revisores y responda a los comentarios.
+
+## Licencia
+
+La mayor parte del contenido de la wiki osu! está bajo la licencia [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md). Esto significa que, en general, los materiales de la wiki se pueden utilizar de manera no comercial (consulta el [resumen rápido](https://creativecommons.org/licenses/by-nc/4.0/)). Algunos contenidos pueden estar cubiertos por diferentes licencias de terceros, que luego se citarán localmente y anularán la licencia global.
```
Comment (agatemosu) [inline wiki/osu!_wiki/es.md:39]:
```suggestion
La mayor parte del contenido de la wiki osu! está bajo la licencia [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md). Esto significa que, en general, los materiales de la wiki se pueden usar de manera no comercial (consulta el [resumen rápido](https://creativecommons.org/licenses/by-nc/4.0/)). Algunos contenidos pueden estar cubiertos por diferentes licencias de terceros, que luego se citarán localmente y anularán la licencia global.
```

File: wiki/osu!_wiki/es.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Ten en cuenta que esto no cubre el uso de «osu!» o marca «ppy» en cualquier software, recurso, publicidad o promoción, ya que está protegida por la ley de marcas. Si necesitas autorización para el uso de estos términos, [contáctenos](mailto:contact@ppy.sh).
```
Comment (agatemosu) [inline wiki/osu!_wiki/es.md:41]:
```suggestion
Ten en cuenta que esto no cubre el uso de «osu!» o marca «ppy» en cualquier software, recurso, publicidad o promoción, ya que está protegida por la ley de marcas. Si necesitas autorización para el uso de estos términos, [contáctanos](mailto:contact@ppy.sh).
```

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**